scapy/layers/ldap.py

```diff
@@ -729,6 +729,8 @@ def answers(self, other):
         return isinstance(other, LDAP) and other.messageID == self.messageID
 
     def mysummary(self):
+        if not self.protocolOp and not self.messageID:
```

Comment:
```suggestion
        if not self.protocolOp or not self.messageID:
```

I think you mean `or` instead of `and`.

Reply:
Yes indeed...